### PR TITLE
Pythonic bind and return rows interface

### PR DESCRIPTION
**Pull request by pfreixes**
Addresses https://github.com/pfreixes/acsylla/issues/34

Implements a new way for binding and getting the results from a row where types are no longer asked, old API is now removed - we are still on Alpha.

For binding values three new methods have been added: `bind`, `bind_by_name`, `bind_list`, `bind_dict` which do not expect any explicit type as part of the signature, types will be inferred behind the scenes by the Driver

For returning column values a the `Value` has been removed, values are directly removed from the `Row.column_value` method. The type of value returned is inferred using a method provided by the CPP driver and returning Python type or an extended Acsylla type.

# Support for UUID

Support for UUID is continued by providing our own UUID types under the public `acsylla.types.UUID` class. Instances of this class will be treated as valid UUID types.

### CODE REVIEW

File: benchmark/cassandra_python_benchmark.py [hunk 1]
```diff
@@ -16,22 +16,39 @@
...
     result = session.execute(statement)
     row = result.one()
     if row is not None:
         _ = row[0]
     return time.monotonic() - start
 
+def write_bind(session, key, str_key):
+    start = time.monotonic()
+    statement = "INSERT INTO test (id, value) values(%s, %s)"
+    session.execute(statement, [key, key])
```
Comment (dmanchon) [inline benchmark/cassandra_python_benchmark.py:38]: [key, key] or [key, str_key] ?

**Review by dmanchon — APPROVED** [review]
LGTM!

**Comment by fruch** [comment]
@pfreixes I really like the direction, would nice to see some example/unittests so it would clear how it would be used.
i.e. if it would be as close to the current python-driver, that would be a great value

**Comment by pfreixes** [comment]
@fruch PR updated with all tests passing, now the unique interface presented for binding is the following one [1]


[1] https://github.com/pfreixes/acsylla/blob/bind-rows-pythonic/acsylla/base.py#L64

**Comment by pfreixes** [comment]
Ready to be reviewed @dmanchon and @fruch it's a little step ahead for having the Beta version

**Pull request merged**